packages/react/examples/use-auth.md

```diff
@@ -0,0 +1,43 @@
+<!-- #region nextjs-01 -->
```

Comment:
In order to use the https://typedoc.org/documents/Tags.__include_.html flag the examples should be in the root of the package, so that relative links work for the files inside `dist`. Because it has to be resolved from `dist` to that folder.